app/src/main/res/layout/activity_edit_history.xml

```diff
@@ -22,11 +23,16 @@
                 android:id="@+id/articleTitleView"
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
-                android:layout_weight="1"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintEnd_toStartOf="@id/compareButton"
                 android:layout_marginEnd="8dp"
+                android:layout_marginTop="8dp"
+                android:layout_marginBottom="8dp"
                 android:lineSpacingExtra="4sp"
                 android:fontFamily="sans-serif"
-                android:maxLines="1"
+                android:maxLines="2"
```

Comment:
Per Robin's comment, he preferred to set the max line to 1 and show the ellipsis if no more spaces.

Reply:
ok, I've changed it back to 1 line for this PR, but I'd be interested to confirm with Robin if this is really desired.
With only 1 line, the title will always be taken up by the "Revision history:" portion, with maybe three characters of the actual title.
![image](https://user-images.githubusercontent.com/1682214/157756017-20691be1-935e-4551-ae3a-41cd43a1c2b0.png)

Reply:
Thanks! Here's what it used to be but Robin thought making it a single line is better 😅
<img src="https://phab.wmfusercontent.org/file/data/7iavym4hx7wwovkdhddc/PHID-FILE-6qhy4ahsoylom6yru245/Screenshot_20220307-111449.png" width="300"/>